web/package.json

```diff
@@ -0,0 +1,5 @@
+{
+  "devDependencies": {
+    "parcel": "^2.12.0"
```

Comment:
Parcel is not really a dev-dependency, it is not used for testing/development, only for making a release bundle, so I'd like to avoid any explicit dependencies to it.

Reply:
Please remove this change from the PR.